saleor/order/utils.py

```diff
@@ -212,3 +213,23 @@ def restock_fulfillment_lines(fulfillment):
         if line.order_line.variant and line.order_line.variant.track_inventory:
             increase_stock(
                 line.order_line.variant, line.quantity, allocate=True)
+
+
+def validate_shipping_method(order):
+    """Checks order for valid shipping method.
```

Comment:
Maybe you could say it `validates` instead of `checks`, so the user explicitly knows what it does is validating an order, otherwise it's a little confusing.